dimagi/pages/templates/sections/blog/mobile_blog_single/blog_contact.html

```diff
@@ -18,7 +18,7 @@ <h4 class="white">
     </div>{# .intro #}
```

Comment:
why are you adding a new template? This is definitely not the solution I explained on call. 

Please add the A/B test decorator for the demo form to the pillar page. Here is an example https://github.com/dimagi/dimagi.com/blob/master/dimagi/pages/views/core.py#L17

Reply:
because, its having diff hubsopt form ID, so I added new template. As a solution we can pass the dynamic formid while calling this template.